quidel_flutest/DETAILS.md

```diff
@@ -0,0 +1,71 @@
+# Quidel Flu Test
+
+### Background
+Starting from 2014-08, we began getting flu test data from Quidel. The data contains a number of features for every test, including localization at 5-digit Zip Code level, a TestDate and StorageDate, patient age, and several identifiers that uniquely identify the device on which the test was performed (SofiaSerNum, the individual test (FluTestNum), and the result (ResultID). Multiple tests are stored on each device and we suspect that the device identifiers could potentially be useful to normalize the results over time. 
+
+### Signal names
+- flu_ag_raw_pct_positive: percent of tests returning positive that day
+- flu_ag_smoothed_pct_positive : same as above, but for the moving average of the most recent 7 days- flu_ag_raw_tests_per_device: average number of tests per active testing device (= device that performed at least one test this period) that day
+- flu_ag_smoothed_tests_per_device: same as above, but for the moving average of the most recent 7 days
+
+### Estimating percent negative test proportion
+Let n be the number of total Flu tests taken over a given time period and a given location (the test result can be negative/positive/invalid). Let x be the number of tests taken with negative results in this location over the given time period. We are interested in estimating the percentage of negative tests which is defined as:
+```
+p = 100 * x / n 
+```
+We estimate p across 3 temporal-spatial aggregation schemes:
+- daily, at the MSA (metropolitan statistical area) level;
+- daily, at the HRR (hospital referral region) level;
+- daily, at the state level.
+We are able to make these aggregations accurately because each test is reported with its 5-digit ZIP code. We do not report estimates for individual counties, as typically each county has too few tests to make the estimated value statistically meaningful.
+
+**MSA and HRR levels**: In a given MSA or HRR, suppose N flu tests are taken in a certain time period, X is the number of tests taken with positive results. If N >= 50, we simply use:
+```
+p = 100 * X / N 
+```
+If N < 50, we lend 50 - N  fake samples from its home state to shrink the estimate to the state's mean, which means:
```

Comment:
IIRC, we only do this in other indicators if N <50 AND > 30. Check if this also applies here